android/java/org/chromium/chrome/browser/crypto_wallet/fragments/EditVisibleAssetsBottomSheetDialogFragment.java

```diff
@@ -78,15 +79,15 @@ public class EditVisibleAssetsBottomSheetDialogFragment extends BottomSheetDialo
     private final boolean mNftsOnly;
     private NetworkInfo mSelectedNetwork;
     private DismissListener mDismissListener;
-    private Boolean mIsAssetsListChanged;
-    private static final String TAG = "EditVisibleAssetsBottomSheetDialogFragment";
+    private boolean mIsAssetsListChanged;
+    private static final String TAG = "EditVisibleAssetsFrag";
```

Comment:
Realized later that Chromium Log probably is not affected by the 23 chars limitation. 
However, I don't see any side effects here, but let me know and in case I'll revert this.

Reply:
presubmit gives an error if it's too long, so all good here.